agave/core/filters.py

```
@@ -2,19 +2,30 @@
 from mongoengine import Q
 
 
+def _ids_filter_value(ids: str | list[str]) -> list[str]:
+    if isinstance(ids, str):
+        return [part.strip() for part in ids.split(',') if part.strip()]
+    return list(ids)
+
+
 def generic_query(query: QueryParams, excluded: list[str] = []) -> Q:
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)